src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -69,6 +69,34 @@ def test_suspended_submission(self):
         submissions_in_session = response.wsgi_request.session[SUBMISSIONS_SESSION_KEY]
         self.assertIn(str(submission.uuid), submissions_in_session)
 
+    @freeze_time("2020-12-11T10:53:19+01:00")
+    @override_settings(LANGUAGE_CODE="en")
+    def test_suspended_submission_not_allowed(self):
+        form = FormFactory.create(suspend_allowed=False)
+        FormStepFactory.create(form=form)
+        step2 = FormStepFactory.create(form=form)
+        submission = SubmissionFactory.create(form=form)
+        SubmissionStepFactory.create(
+            submission=submission, form_step=step2, data={"foo": "bar"}
+        )
+        self._add_submission_to_session(submission)
+        endpoint = reverse("api:submission-suspend", kwargs={"uuid": submission.uuid})
+
+        response = self.client.post(endpoint, {"email": "hello@open-forms.nl"})
+        self.assertEqual(response.status_code, 403)
+        response_data = response.json()
+        self.assertEqual(
+            response_data["type"], "http://testserver/fouten/PermissionDenied/"
+        ),
+        self.assertEqual(response_data["code"], "permission_denied"),
+        self.assertEqual(
+            response_data["title"], "You do not have permission to perform this action."
+        ),
+        self.assertEqual(response_data["status"], 403),
+        self.assertEqual(
+            response_data["detail"], "Suspending this form is not allowed."
+        ),
```

Comment:
```suggestion
        response = self.client.post(endpoint, {"email": "hello@open-forms.nl"})
        
        self.assertEqual(response.status_code, 403)
        error = response.json()["invalidParams"][0]
        self.assertEqual(error["code"], "permission_denied")
```

1. The error structure you're testing does not adhere to the documented error structure in the API spec
2. The status code and error code are most important, the error detail message is "fluff", no need to test for that and deal with translation machinery.